sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/FileClientBuilder.java

```diff
@@ -60,16 +70,30 @@
  * @see FileAsyncClient
  * @see SharedKeyCredential
  */
-@ServiceClientBuilder(serviceClients = {FileClient.class, FileAsyncClient.class, DirectoryClient.class,
-    DirectoryAsyncClient.class})
-public class FileClientBuilder extends BaseFileClientBuilder<FileClientBuilder> {
-
+@ServiceClientBuilder(serviceClients = {
+    FileClient.class, FileAsyncClient.class,
+    DirectoryClient.class, DirectoryAsyncClient.class
+})
+public class FileClientBuilder {
     private final ClientLogger logger = new ClientLogger(FileClientBuilder.class);
 
+    private String endpoint;
+    private String accountName;
```

Comment:
What's the reason to have accountName here?

Reply:
We added being able to get the account name from every client. It is just another way to help move around and understanding what is being used.